package.json

```diff
@@ -26,7 +26,7 @@
     "@bcrs-shared-components/court-order-poa": "2.1.3",
     "@bcrs-shared-components/date-picker": "1.2.5",
     "@bcrs-shared-components/detail-comment": "1.1.1",
-    "@bcrs-shared-components/enums": "1.0.27",
+    "@bcrs-shared-components/enums": "1.0.36",
```

Comment:
OK, I see the other shared imports are WAY out of date. I'm OK to leave it for now, since it will require some looking at code to see what changed and then testing the changes. Please create a new ticket (Tech Debt) to do this.

Reply:
Deal, will do.

Reply:
What's the new ticket's number?

Reply:
https://github.com/bcgov/entity/issues/15764